frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -166,7 +166,7 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
               <TextFilter
                 value={textFilter}
                 label="by name or description"
-                onChange={(e) => this.setState({ textFilter: e.target.value })}
+                onChange={(e) => this.setState({ textFilter: e })}
```

Comment:
If `e` is not an event, we should give it a different name.

```suggestion
                onChange={(textFilter: string) => this.setState({ textFilter })}
```